package/gluon-wan-dnsmasq/files/lib/gluon/wan-dnsmasq/interface.d/wan-bridge

```diff
@@ -0,0 +1,2 @@
+wan6
+wan
```

Comment:
Let's make the file name match the package name.

Reply:
Hmm, and should we prepend a number to the filename to specify a precedence?

Reply:
FWIW, this specifies the order in which nameservers are added to resolv.conf. So this sounds like a good idea.